main/src/ca/uwaterloo/flix/api/effectlock/serialization/STC.scala

```diff
@@ -0,0 +1,112 @@
+package ca.uwaterloo.flix.api.effectlock.serialization
+
+sealed trait STC
+
+object STC {
+
+  case object Void extends STC
+
+  case object AnyType extends STC
+
+  case object Unit extends STC
+
+  case object Null extends STC
+
+  case object Bool extends STC
+
+  case object Char extends STC
+
+  case object Float32 extends STC
+
+  case object Float64 extends STC
+
+  case object BigDecimal extends STC
+
```

Comment:
@magnus-madsen we could remove the wrapper object `STC` so the objects are serialized to their names rather than `STC$Name`. E.g., `Void` would be serialized to `Void` instead of `STC$Void`

Reply:
I think that is reasonable.